.circleci/config.yml

```diff
@@ -168,34 +169,25 @@ jobs:
       - image: *postgres_image
       - image: *redis_image
 
-  tests_postgres_rails_6_1:
-    <<: *tests_postgres
+  tests_ruby_2_7_rails_6_1_postgres:
+    <<: *tests_ruby_2_7_rails_7_0_postgres
     environment:
       <<: *postgres_environment
       RAILS_VERSION: '~> 6.1.0'
 
-  tests_mysql: &tests_mysql
```

Comment:
I think this contained some minimal configuration for MySQL tests on Ruby 2.7 (but it was not run anywhere). Maybe we can also review the missing configurations (e.g. Rails 6.1 + Ruby 3.0) to see whether it makes sense to cover some of them.

Reply:
Technically we could, but I'm wondering if it makes sense to complicate it further. My thinking was: for now the key configurations would be Ruby 2.7 + Rails 6.1 (for older applications that can't easily upgrade ruby/rails and Ruby 3.0 + Rails 7 (for new projects). Additionally, running the suite on MySQL would additionally show us if there are any queries that are incompatible with MySQL.

What do you think about it?

Reply:
Agreed - it's sufficient.